test.sh

```diff
@@ -18,6 +18,7 @@ function usage()
     echo "Usage: $0 [-m MARKEXPR] [-k EXPRESSION] [-p PATH] [-s] all|<framework-name>"
     echo "-m passed to pytest directly [default -m \"sanity and not azure\"]"
     echo "-k passed to pytest directly [default NONE]"
+    echo "-f module to test passed directly to pytest [default test_*.py]"
```

Comment:
"pytest -k stringexpr # only run tests with names that match the
                      # "string expression", e.g. "MyClass and not method"
                      # will select TestMyClass.test_something
                      # but not TestMyClass.test_method_simple"

Seems like since the tests in secrets are prefixed with `test_secrets`, we could use -k in this scope. But we don't seem to be following that convention (yet) so that functionality doesn't seem to target all tests at the module-level if not all the tests are prefixed wrt module.